zsvjmp-ppc64le.S

```diff
@@ -0,0 +1,61 @@
+
+# Copyright 2018 Gustavo Romero, Rogerio Cardoso, Breno Leitao, IBM Corporation.
+
+	.file "zsvjmp.s"
+	.abiversion 2
+	.section ".text"
+	.align 2
+	.globl zsvjmp_
+	.type zsvjmp_, @function
+
```

Comment:
This is a really long comment. I suggest to change it to something like:

> Use VRSAVE (unused in Linux) and stack reserved area (SP+12) to save the return address (LR)  which is lost after `bl __sigsetjmp`. External functions should be called using a  `bl/nop` pair, which is turned into a  `plt_call stub` by the linker, ppc64 doesn't  support `b function@plt` anymore. 

And write a comment before line [56](https://github.com/leitao/zsvjmp/pull/2/commits/c4722c4e174890d5fa7e830cd92a2154292510ad#diff-503b9d4c8e36e4be9f5b6278b6a6eb84R56) about why restore VRSAVE:

> VRSAVE is saved on the buffer. Some few libc can use it as a boolean as an optimization so it needs to be restored to -1 (VRSAVE value is always -1).